spec/ig-mcode-config.json

```diff
@@ -1,26 +1,43 @@
 {
-    "projectName": "HL7 FHIR Implementation Guide: minimal Common Oncology Data Elements (mCODE), Release 0.7 - US Realm (Draft for Comment 1)",
+    "projectName": "HL7 FHIR Implementation Guide: minimal Common Oncology Data Elements (mCODE), Release 0.8 - US Realm (Draft for Comment 1)",
     "projectShorthand": "oncocore",
     "projectURL": "http://hl7.org/fhir/us/projectURL/",
     "fhirURL": "http://hl7.org/fhir/us/shr/",
     "entryTypeURL": "http://hl7.org/fhir/us/entryTypeURL/spec/",
     "fhirTarget": "FHIR_DSTU_2",
+    "contentProfile": "cp-mcode",
```

Comment:
Falling in line with my previous comment, please change this line to `"contentProfile": "mcode_cp.txt"` and this should be as expected by the content profile importer.

Reply:
If you do end up choosing not to rename the file, which I'll leave up to you, you do at least have to add the `.txt` extension to this field. For example, `"contentProfile: "cp-mcode.txt"`.

Reply:
Thanks, I added .txt and kept cp-mcode